src/views/details/details.js

```diff
@@ -0,0 +1,97 @@
+myApp.DetailsPage = function(i) {
+    const evData = localStorage.getItem("evaluationsData");
+    const tableData = JSON.parse(evData);
+    this.markup = `
+    ${myApp.DetailsTable(tableData)}
+    `
+}
+
+myApp.DetailsTable = function(tableData) {
+    return `
+    
+             <p>Candidate Info:</p>
+             <table class="modal-table">
+                 <tr>
+                     <td class="modal-table-data">Candidate Name</td>
+                     <td class="modal-table-data">${tableData[0].candidateInfo[0].value}</td>
```

Comment:
I find that this approach of multiple nested arrays may quickly and easily become burdensome to maintain in the long run. Wherever possible, I'd recommend using objects with properly named properties instead of variable length arrays, each of the array's elements representing a particular piece of information pertaining to the whole.